lib/internal/modules/esm/loader.js

```diff
@@ -297,15 +297,15 @@ class ModuleLoader {
     return module.getNamespace();
   }
 
-  register(specifier, parentUrl) {
+  register(specifier, parentUrl, data, transferList) {
```

Comment:
```suggestion
  /**
   * @see {CustomizedModuleLoader.register}
   */
  register(specifier, parentUrl, data, transferList) {
```

I think this syntax is right

Reply:
According to https://jsdoc.app/tags-see.html, it should be
```suggestion
  /**
   * @see {@link CustomizedModuleLoader.register}
   */
  register(specifier, parentUrl, data, transferList) {
```

Not sure if it matters though as we are not generating our docs with the JSDoc comments.

Reply:
VS Code (and possibly others) consumes them :)